### ISSUE TITLE
Create a reference to http.createServer

### ISSUE THREAD

**Issue by michaeljcalkins** [adonisjs/adonis-framework#175]
Could you make a reference to the created server?
https://github.com/adonisjs/adonis-framework/blob/9a9f3d45147740d6a0ff908efb5f23a0862e9151/src/Server/index.js#L280

I wanted to use this for my game but need to be able to do something like:
```javascript
var server = http.createServer(app);

/**
 * Listen on provided port, on all network interfaces.
 */

server.listen(port);
server.on('error', onError);
server.on('listening', onListening);

/**
 * SocketIO Connection
 */
var io = app.io
io.attach(server)
```

**Comment by RomainLanz** [comment]
There's already a pull-request who add that to the core.

https://github.com/adonisjs/adonis-framework/pull/165

**Comment by thetutlage** [comment]
Closed, merged in #165
